src/mlpack/tests/main_tests/nbc_test.cpp

```diff
@@ -331,4 +331,111 @@ BOOST_AUTO_TEST_CASE(NBCIncrementalVarianceTest)
   CheckMatrices(output_probs, CLI::GetParam<arma::mat>("output_probs"));
 }
 
+/**
+  * Ensuring that the parameter 'output' and the parameter 'predictions' give
+  * the same output.
+ **/
+// The following test case is to check whether the old parameter 'output' and
+// the new parameter 'predictions' give the same output.
+// This test case will be removed in mlpack4
+// when the deprecated parameter: 'output' is removed.
```

Comment:
Some of this comment is redundant and you can put it all in the block up above, instead of having some as `/* */` and some as `//`.